content/news/posts/2024/PSDevCon 2024-PrestaShop-UIKit.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: PSDevCon 2024 - PUIK (PrestaShop UI Kit) - The new UI kit from PrestaShop
+subtitle: Explore a summary of the PUIK presentation delivered at the second PrestaShop Developer Conference
+date: 2024-12-12
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags: [news, event]
+---
+
+During the second PrestaShop Developer Conference on November 6, 2024, Matthias Goudjil introduced PUIK (PrestaShop UI Kit). This blog post provides a written summary of his presentation. During the talk, participants gained a clearer understanding of PrestaShop's UI Kit, based on the new PrestaShop Design System. 
+Learn how this library aims to provide a complete set of reusable components for the entire PrestaShop ecosystem. The GitHub repository is [available here](https://github.com/PrestaShopCorp/puik).
+
+## Introduction
+
+### Initial objective: create a library of UI components in Vue.js
+
+PUIK is a library of UI components developed in Vue 3. It is an open source project under the MIT license, not to be confused with the [UI kit]( https://github.com/PrestaShop/prestashop-ui-kit) of PrestaShop, which is integrated into the PrestaShop solution and uses Bootstrap for styling and jQuery for interactivity.
+The initial objective was to provide the same components to ensure consistency and address the need for design uniformity across the products and services offered within the PrestaShop ecosystem.
+
+The choice of Vue.Js was driven by its widespread use among the teams. Another major reason is that the PrestaShop solution itself incorporates this framework: some back-office pages and modules already use Vue.Js.
+
+### Architecture version 2
+
+The current version of PUIK at the time of writing this article is 2.3.0.
+
+The first version, while still maintained for critical bugs, will eventually be deprecated once all teams have migrated to the second version.
```

Comment:
```suggestion
While still maintained for critical bugs, the first version will eventually be deprecated.
```

Information about PS teams having to migrate to a new version is not relevant imho.